### PR TITLE
Forge deploy testing fix

### PR DESCRIPTION
**Pull request by wighawag**

### CODE REVIEW

File: utils/BaseScript.sol [hunk 1]
```diff
@@ -11,7 +11,7 @@ abstract contract BaseScript is DeployScript {
     Constants internal immutable constants = new Constants(vm);
     bool internal testing;
 
-    function deployer() public view returns (address) {
+    function _deployer() public view returns (address) {
```
Comment (wighawag) [inline utils/BaseScript.sol:14]: Oh wait, this was done to make it compile
But this is not what you want as places where they were getting this, they will now get the forge-deploy's deployer contract address

**Review by 0xCalibur — APPROVED** [review]

**Comment by 0xCalibur** [comment]
super helpful, thanks a lot!

File: utils/BaseScript.sol (hunk 1 above)
Comment (0xCalibur) [reply utils/BaseScript.sol:14]: I'll just remove it, we never use it anyway :)

**Comment by wighawag** [comment]
> super helpful, thanks a lot!

Thanks too, it help a lot to have a real project to test it.

**Pull request merged**